From ecashin@coraid.com Thu Jan 19 11:11:56 2006
Message-ID: <49d97c1e1c57eeb1017586aef4372f8c@coraid.com>
Date: Thu, 19 Jan 2006 13:46:29 -0500
From: "Ed L. Cashin" <ecashin@coraid.com>
CC: ecashin@coraid.com, Greg K-H <greg@kroah.com>
Subject: [PATCH 2.6.15-git9] aoe [7/8]: update driver compatibility string

The aoe driver is not compatible with 2.6 kernels older
than 2.6.2.

Signed-off-by: "Ed L. Cashin" <ecashin@coraid.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/block/aoe/aoemain.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/block/aoe/aoemain.c
+++ gregkh-2.6/drivers/block/aoe/aoemain.c
@@ -11,7 +11,7 @@
 
 MODULE_LICENSE("GPL");
 MODULE_AUTHOR("Sam Hopkins <sah@coraid.com>");
-MODULE_DESCRIPTION("AoE block/char driver for 2.6.[0-9]+");
+MODULE_DESCRIPTION("AoE block/char driver for 2.6.2 and newer 2.6 kernels");
 MODULE_VERSION(VERSION);
 
 enum { TINIT, TRUN, TKILL };
